settings.py

```diff
@@ -477,7 +477,7 @@
 SUBSCRIPTION_CHANGE_EMAIL = 'accounts+subchange@example.com'
 INTERNAL_SUBSCRIPTION_CHANGE_EMAIL = 'accounts+subchange+internal@example.com'
 BILLING_EMAIL = 'billing-comm@example.com'
-INVOICING_CONTACT_EMAIL = 'billing-support@example.com'
+INVOICING_CONTACT_EMAIL = 'accounts@example.com'
```

Comment:
This is @example.com, not @dimagi.com. Does it need to change?

Reply:
Hi Daniel, I want to keep this file a "mirror" for how we actually setup those email.